tests/manage/pv_services/test_overprovision_level_policy_control.py

```diff
@@ -0,0 +1,222 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import defaults
+from ocs_ci.ocs.resources.storage_cluster import verify_storage_cluster
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier1,
+    bugzilla,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@bugzilla("2024545")
+@pytest.mark.polarion_id("OCS-XYZ")
+class TestOverProvisionLevelPolicyControl(ManageTest):
+    """
+    Test OverProvision Level Policy Control
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            log.info("Delete overprovisionControl from storage cluster yaml file")
+            storagecluster_obj = OCP(
+                resource_name=constants.DEFAULT_CLUSTERNAME,
+                namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+                kind=constants.STORAGECLUSTER,
+            )
+            params = '{"spec": {"overprovisionControl": []}}'
+            storagecluster_obj.patch(
+                params=params,
+                format_type="merge",
+            )
+            log.info("Verify storagecluster on Ready state")
+            verify_storage_cluster()
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.parametrize(
+        argnames=["sc_name", "sc_type"],
+        argvalues=[
+            pytest.param(*[constants.CEPHBLOCKPOOL_SC, constants.CEPHBLOCKPOOL]),
+            pytest.param(*[constants.CEPHFILESYSTEM_SC, constants.CEPHFILESYSTEM]),
+            # https://bugzilla.redhat.com/show_bug.cgi?id=2120121
+            # pytest.param(*["sc-test-blk", constants.CEPHBLOCKPOOL]),
+            # https://bugzilla.redhat.com/show_bug.cgi?id=2120121
+            # pytest.param(*["sc-test-fs", constants.CEPHFILESYSTEM]),
```

Comment:
Is this new BZ created by you during the TC validation? if so can we add Bugzilla skip marker?

Reply:
It was not a bug.. I deleted the secret